app/views/projects/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h2> Project name: <%=@project.name %></h2>
+  <p>Material: <%=@project.material %></p>
+  <p>Challenge Theme: <%= @theme %></p>
+  <p>Number of Contestants: <%= @project.contestants.count %></p>
+  <p>Average Contestant Experience: <%= @project.contestants.avg_exp %></p>
+
+<h3> Add a contestant to this project </h3>
+  <%= form_with url: "/contestants/new", method: :post, data: { turbo: false } do |f| %>
```

Comment:
We aren't creating a new contestant here. We are adding an existing contestant to an existing project. 
```
When I visit a project's show page
I see a form to add a contestant to this project
When I fill out a field with an existing contestants id
```
So your form url would want to point to a join controller to create a new relationship between the two:
```ruby
<%= form_with url: "/project/#{@project.id}/contestants", method: :post, local: true do |form| %>
  <%= form.label :contestant_id %>
  <%= form.number_field :contestant_id %>

  <%= form.submit "Add Contestant To Project" %>
<% end %
```
So your route would look something like:
```ruby
post "/project/:project_id/contestants", to: "contestant_projects#create"
```